WordPress/Classes/ViewRelated/Blog/Blog Dashboard/BlogDashboardViewController+QuickActions.swift

```diff
@@ -0,0 +1,52 @@
+import Foundation
+import WordPressShared
+
+// MARK: - Quick Actions
+
+extension BlogDashboardViewController {
+
+    func showStats() {
+        trackQuickActionsEvent(.statsAccessed)
+
+        let controller = StatsViewController()
+        controller.blog = blog
+        controller.navigationItem.largeTitleDisplayMode = .never
+        showDetailViewController(controller, sender: self)
+
+        QuickStartTourGuide.shared.visited(.stats)
```

Comment:
I wonder if we can create a static method on `StatsViewController` that: given a `Blog` and a `ViewController`, it presents itself and also handle the `QuickStartTourGuide.shared.visited(.stats)`.

Otherwise, we'll have to make use of the `QuickStartTourGuide.shared.visited(.stats)` in every place we present it.

Ps.: this is to "finish" the tour for checking stats, right?

Reply:
Fixed! edc0ac1

> Ps.: this is to "finish" the tour for checking stats, right?
Yep  -- may have been a bit too early to add this before tackling the quick start task though